src/Form/Select.svelte

```diff
@@ -54,24 +84,49 @@
     bottom: 0 !important;
     position: absolute !important;
     pointer-events: none !important;
-    padding-left: 0.5rem !important;
     align-items: center !important;
     display: flex !important;
+    box-sizing: border-box;
   }
+
+  .editable {
+    opacity: 0;
+  }
+
+  .editable-pointer {
+    border-style: solid;
+    border-width: 0 0 0 1px;
+    border-color: var(--grey-4);
+    padding-left: var(--spacing-m);
+  }
+
 </style>
 
-<div class="relative">
+<div class="relative" class:disabled class:secondary class:outline>
   <select
     {name}
     class:thin
     class:secondary
-    class:outline
+    class:editable
     {disabled}
     on:change
+    on:blur
     bind:value>
     <slot />
   </select>
-  <div class="pointer">
+  {#if editable}
+    <input  
+      class:thin 
+      class:secondary 
+      class:disabled
+      on:change={updateValue("change")}
```

Comment:
I think we might be able to get rid of the event dispatcher and curried function and just bind the input the same way we are with the `select` here.

```
on:input
on:blur
bind:value
```

Reply:
Yes - it was like this... but was not working... the blur event was not firing (?)

Reply:
Ah - I think this might be because the `blur` event doesn't bubble.

Reply:
updated so only on:blur does a `dispatch`